pennylane/qchem/vibrational/pes_generator.py

```diff
@@ -822,7 +822,8 @@ def vibrational_pes(
 
     comm = MPI.COMM_WORLD
     rank = comm.Get_rank()
-    molecule, scf_result = optimize_geometry(molecule, method)
+    molecule = optimize_geometry(molecule, method)
```

Comment:
The output should be an array not the molecule object, then you can reconstruct the molecule object with the new geometry here and pass it to _single_point